### PR TITLE
💄✨ Add Co-Author field to Blog Posts

### PR DESCRIPTION
**Pull request by joshbermanssw**
As per #2107 we wanted to be able to acknowledge users when they are also 'checking' blog posts (as this can sometimes be a lengthy process 😅) 

**Changes:**
- Added a string list called co-Authors to the blog schema 
- Added the co-author field to the getExpandedBlogPosts gql query 
- Rendered the co-authors field on both the `page-index.tsx` file and `[slug].tsx` file for blogs 

Screenshot 2024-09-11 at 12 12 58 PM 1

**Figure: 📄 Co-authors displayed on a blog post**

Screenshot 2024-09-11 at 12 28 35 PM

**Figure: 📄 Co-authors displayed on minimised blog post /blog page**

### CODE REVIEW

File: content/blog/TinaCMS-sponsoring-the-Copenhagen-Developers-Festival.mdx [hunk 1]
```diff
@@ -3,13 +3,16 @@ title: TinaCMS sponsoring the Copenhagen Developers Festival
 date: '2024-08-13T14:00:00.000Z'
 last_edited: '2024-08-13T14:00:00.000Z'
 author: Landon Maxwell
+coAuthors:
```
Comment (bradystroud) [inline content/blog/TinaCMS-sponsoring-the-Copenhagen-Developers-Festival.mdx:6]: Coauthors is one word, sometimes hypenated

```suggestion
coauthors:
```

or 

```suggestion
co_authors:
```

File: tina/collectionsSchema/blogs.tsx [hunk 2]
```diff
@@ -43,6 +43,12 @@ export const blogsCollection = {
       name: 'author',
       label: 'Author',
     },
+    {
+      type: 'string',
+      name: 'coAuthors',
```
Comment (bradystroud) [inline tina/collectionsSchema/blogs.tsx:48]: same as above

File: tina/queries/post.gql [hunk 3]
```diff
@@ -13,6 +13,7 @@ query getExpandedPostDocument($relativePath: String!) {
     date
     last_edited
     author
+    coAuthors
```
Comment (bradystroud) [inline tina/queries/post.gql:16]: same as above

**Review by bradystroud — CHANGES_REQUESTED** [review]

**Comment by joshbermanssw** [comment]
awaiting designer

**Pull request closed without merging**